unittests/framework/platforms/node/raid/test_raid.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import json
+import os
+import sys
+import unittest
+from unittest.mock import Mock, mock_open, patch
+
+PROJECT_ROOT = "/".join(os.path.abspath(__file__).split("/")
```

Comment:
No need for this. pytest will read the paths from `unittests/conftest.py`

Reply:
Done